pkgs/by-name/va/vaporizer2/package.nix

```diff
@@ -0,0 +1,78 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  juce,
+  pkg-config,
+  alsa-lib,
+  fftwFloat,
+  fontconfig,
+  freetype,
+  libGL,
+  libX11,
+  libXcursor,
+  libXext,
+  libXinerama,
+  libXrandr,
+  libjack2,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "vaporizer2";
+  version = "3.5.0-unstable-2024-09-14";
+
+  src = fetchFromGitHub {
+    owner = "VASTDynamics";
+    repo = "Vaporizer2";
+    rev = "1c56c4be304255f1c397dad725ea784f15a55ef5";
+    hash = "sha256-/J0HqiXc84JEIsEyb1MlzAqpmJZEe8jP07fEcJSEZz8=";
+    fetchSubmodules = true;
+  };
+
+  strictDeps = true;
+
+  nativeBuildInputs = [
+    cmake
+    juce
+    pkg-config
+  ];
+
+  buildInputs = [
+    alsa-lib
+    fftwFloat
+    fontconfig
+    freetype
+    libGL
+    libX11
+    libXcursor
+    libXext
+    libXinerama
+    libXrandr
+    libjack2
+  ];
+
+  cmakeFlags = [
+    (lib.cmakeFeature "USE_SYSTEM_JUCE" "ON")
+    (lib.cmakeFeature "CMAKE_BUILD_TYPE" "Release")
```

Comment:
1. Use `cmakeBuildType` for settings this.
2. `cmakeBuildType` should default to `"Release"`, so this should already be getting set.

```suggestion
```